vhdbuilder/packer/post-install-dependencies.sh

```diff
@@ -38,11 +41,13 @@ echo -e "=== Installed Packages Begin\n$(listInstalledPackages)\n=== Installed P
 echo "Disk usage:" >> ${VHD_LOGS_FILEPATH}
 df -h >> ${VHD_LOGS_FILEPATH}
 
-
-# warn at 75% space taken
-[ -s $(df -P | grep '/dev/sda1' | awk '0+$5 >= 75 {print}') ] || echo "WARNING: 75% of /dev/sda1 is used" >> ${VHD_LOGS_FILEPATH}
-# error at 99% space taken
-[ -s $(df -P | grep '/dev/sda1' | awk '0+$5 >= 99 {print}') ] || exit 1
+# check the size of the OS disk after installing all dependencies: warn at 75% space taken, error at 99% space taken
+os_disk=$(readlink -f /dev/disk/azure/root-part1)
+used_blocks=$(df -P | grep -w "${os_disk}" | awk '{print $3}')
+usage=$(echo "scale = 2; (${used_blocks} / ${MAX_BLOCK_COUNT}) * 100" | bc)
```

Comment:
mehhhhh. I have somewhat strong feelings against bc but if there's no sane way without it I'm okay with it. the actual diff looks good.

Reply:
awk to the rescue!